parachains/runtimes/assets/statemine/src/lib.rs

```diff
@@ -596,6 +610,9 @@ construct_runtime!(
 		// The main stage.
 		Assets: pallet_assets::{Pallet, Call, Storage, Event<T>} = 50,
 		Uniques: pallet_uniques::{Pallet, Call, Storage, Event<T>} = 51,
+
+		#[cfg(feature = "state-trie-migration")]
+		StateTrieMigration: pallet_state_trie_migration = 52,
```

Comment:
Just for tidyness I'd leave the 50s clear for new pallets related to the core functionality of Statemine. Could this be `5` and just go with the system stuff?
```suggestion
		StateTrieMigration: pallet_state_trie_migration = 70,
```

Reply:
I did switch to 70 as in the suggested change, let me know if it should be `5` (the numbering scheme is something I am not familiar with).